### PR TITLE
Fix the nits found in the testcases of `PodTopologySpread`

### PR DESCRIPTION
**Pull request by chendave**
Signed-off-by: Dave Chen <dave.chen@arm.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by chendave** [comment]
/sig scheduling

cc @Huang-Wei

File: pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go [hunk 1]
```diff
@@ -1488,9 +1488,9 @@ func TestMultipleConstraints(t *testing.T) {
 		},
 		{
 			// 1. to fulfil "zone" constraint, incoming pod can be placed on zone2 (node-x or node-y)
-			// 2. to fulfil "node" constraint, incoming pod can be placed on node-b or node-x
+			// 2. to fulfil "node" constraint, incoming pod can be placed on node-a, node-b or node-x
 			// intersection of (1) and (2) returns node-x
-			name: "Constraints hold different labelSelectors, spreads = [1/0, 1/0/0/1]",
+			name: "Constraints hold different labelSelectors, spreads = [1/0, 0/0/0/1]",
```
Comment (Huang-Wei) [inline pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go:1493]: Pod `p-a1` is located on node-a.

Comment (chendave) [reply pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go:1493]: Due to the constraints is defined as this,
```
SpreadConstraint(1, "node", v1.DoNotSchedule, st.MakeLabelSelector().Exists("bar").Obj()).
```
and this pod is defined as this,
```
st.MakePod().Name("p-a1").Node("node-a").Label("foo", "").Obj(),
```
constraint selector doesn't match with the pod label,

Double confirmed by,

```
(dlv) p s.TpPairToMatchNum
map[k8s.io/kubernetes/pkg/scheduler/framework/plugins/podtopologyspread.topologyPair]*int32 [
        {key: "zone", value: "zone2"}: *0,
        {key: "node", value: "node-y"}: *1,
        {key: "zone", value: "zone1"}: *1,
        {key: "node", value: "node-b"}: *0,
        {key: "node", value: "node-x"}: *0,
        {key: "node", value: "node-a"}: *0,
]
```

Comment (Huang-Wei) [reply pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go:1493]: I know, but that comment just literally describes the fact how the pods are located across the cluster - it's not related with incoming pod's constraints.

Comment (chendave) [reply pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go:1493]: okay, I will revert this line back.

Comment (chendave) [reply pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go]: Done

**Pull request merged**